### DISCUSSION TITLE
Errors: Unexpected token M in JSON at position 875

### DISCUSSION THREAD

**Discussion by Scotsky** [rern/rAudio#588] [Fixed Issues]
Hi,

After have used Runeaudio for a few years now, I was tryíng to make the switch to r.audio.
The setup went as a charm, but when I was adding my NAS files, and made a mistake, I got the above error.
I installed the image 3 times over again, and everytime when making a mistake in adding the location/share of my music, I come back to this kind of error.
I have access with SSH to the Pi, but don't know were to edit, or correct that wrong source path, or resset it.
The error occurs only when I acces the system tab of r.audio.

Everything else works great.
My setup is a Raspberry Pi 3B+ and a HiFiberry DAC+
The connection can be wired, or LAN, it doesn't matter.

I hope someone put me in the right direction, because, I don't have any clue were to look for it.

Here is the latest error, I got, you can see, I made more then one mistake with addingen the source path:
I changed the ip-adres with x.x.x.x.
-----
{ "page" : "system" , "cpuload" : "1.12 0.37 0.13" , "cputemp" : 27.3 , "startup" : "6s (kernel) + 11s (userspace) " , "throttled" : "0x0" , "time" : "14:49:35 2021-11-01" , "timezone" : "Europe/Amsterdam" , "uptime" : "1m" , "uptimesince" : "2021-11-01 14:48" , "audioaplayname" : "hifiberry-dacplus" , "audiooutput" : "HiFiBerry DAC+" , "bluetooth" : false, "bluetoothconf" : [ true, false ] , "firmware" : "20211004-1" , "hostapd" : false, "hostname" : "rAudio" , "kernel" : "5.10.74-1-raspberrypi4-ARCH aarch64" , "lcd" : false, "lcdchar" : false , "lcdcharaddr" : "0x27 0x3F" , "lcdcharconf" : [ 20,"A00","i2c","0x27","PCF8574",15,18,16,21,22,23,24,false ] , "list" : [ {"icon":"microsd","mountpoint":"/","mounted":true,"source":"/dev/mmcblk0p2","size":"2.9GB/15GB"},{"icon":"networks","mountpoint":"/mnt/MPD/NAS/Music - V.A","mounted":false,"source":"//x.x.x.x/\▮Music - V.A"},{"icon":"networks","mountpoint":"/mnt/MPD/NAS/Music - V.A","mounted":false,"source":"//x.x.x.x/music\Music - V.A"},{"icon":"networks","mountpoint":"/mnt/MPD/NAS/Music - V.A","mounted":false,"source":"//x.x.x.x/music/Music - V.A"},{"icon":"networks","mountpoint":"/mnt/MPD/NAS/Music - V.A","mounted":false,"source":"//x.x.x.x//Music/Music - V.A"},{"icon":"networks","mountpoint":"/mnt/MPD/NAS/Music - V.A","mounted":false,"source":"//x.x.x.x/Music/Music - V.A"} ] , "lcdmodel" : "tft35a" , "mpdoled" : false , "mpdoledconf" : 6 , "online" : true , "ntp" : "pool.ntp.org" , "powerbutton" : false, "powerbuttonconf" : [ 5,40,5 ] , "relays" : false , "rpimodel" : "Raspberry Pi 3 Model B Plus Rev 1.3" , "soc" : "BCM2837B0" , "soccpu" : "4" , "socram" : "952MiB" , "socspeed" : "1400" , "soundprofile" : false , "soundprofileconf" : [ 18000000 ,60 ,1500 ,1000 ] , "version" : "1" , "versionui" : 20211022 , "vuled" : false , "vuledconf" : [ 14,15,18,23,24,25,8 ] , "wlan" : true , "wlanconf" : [ "00" , false ] , "wlanconnected" : true}

**Comment by kan-ibal** [comment]
Check a script for system tab in /srv/http/bash/system-data.sh, but at position 875 there is "source":"//x.x.x.x/\▮Music - V.A"}, so it looks like you have to umount  this source from /mnt/MPD/NAS/Music and thats all.

**Reply by Scotsky** [reply]
Thanks for quick answer.
With your information, I found that the info was stored in /etc/fstab 
I deleted the wrong entree's and it works now.
(Now to play with the correct syntax to add a NAS path, but that won't be a big problem.)

Thanks for big help :-)
